houston/mission.py

```diff
@@ -20,21 +21,22 @@ class Mission(object):
     configuration = attr.ib(type=Configuration)
     environment = attr.ib(type=Environment)
     initial_state = attr.ib(type=State)
-    commands = attr.ib(type=List[Command])  # FIXME
+    commands = attr.ib(type=Union[List[Command], Tuple[Command]],
```

Comment:
The `type` property specifies the type of the values that are used by that attribute. To avoid misleading the static analysis, `type` should be `Tuple[Command]` in this case.